src/admin/admin.controller.ts

```diff
@@ -47,10 +47,11 @@ export class AdminController {
   @Get('/reports')
   async getAllReports(
     @Query('isDeleted', ParseBoolPipe) isDeleted: boolean,
+    @Query('type') type: string,
   ): Promise<FullReportDto[]> {
     return isDeleted
       ? await this.adminService.getAllDeletedReports()
```

Comment:
It's strange that when the `isDeleted` parameter is set to true, there is no observable change in the response. This anomaly arises due to the fact that the said parameter is not being forwarded to the `adminService.getAllDeletedReports` function.